app.conf.ctmpl

```diff
@@ -0,0 +1,21 @@
+{{with $environment := env "ENVIRONMENT"}}
+{{with $dataSource := vault (printf "secret/dsde/avram/%s/dbcDataSource" $environment)}}
+{{with $googleProject := env "GOOGLE_PROJ"}}
```

Comment:
`googleProject` and `serviceVersion` are unused here.

Reply:
Yep - I'm removing those - I was using those to test config rendering before I rebased on top of your stuff